app/src/main/java/edu/agh/susgame/front/gui/components/menu/components/createlobby/CreateLobbyView.kt

```diff
@@ -98,23 +89,27 @@ fun CreateLobbyView(
                     )
                     CreateGameComp(
                         gameName = gameName,
-                        gamePin = gamePin,
+                        gamePin = if (isGamePinEnabled) LobbyPin(gamePinInputValue) else null,
                         selectedNumberOfPlayers = selectedNumberOfPlayers,
                         lobbyService = lobbyService,
                         navController = navController
                     )
                 }
-                Column(modifier = Modifier.fillMaxSize().weight(4f),
+                Column(
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .weight(2f),
                     verticalArrangement = Arrangement.Center,
                     horizontalAlignment = Alignment.CenterHorizontally
                 ) {
-
+                    // TODO Add some content here
```

Comment:
Stworzysz tickecik?

Reply:
czekam na nowe mapy od andrzeja, jak tylko zrobi to je ogarnę szybko,